modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -0,0 +1,85 @@
+[[quickstart-sap-setup]]
+= Preparing Clients
+
+Your SAP cluster requires several client systems.
+Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready on your {productname} Server.
+You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+
+One of the key features of SAP is high availability of the cluster.
+Every component within an SAP cluster has redundancy and failover protection.
+When you are preparing your clients, ensure you have enough hardware and infrastructure to allow for this.
+For more information about hardware requirements, see https://documentation.suse.com/sles-sap/15-SP1/html/SLES4SAP-quick/cha-s4s-plan.html#sec-s4s-hardware[].
+
+For more information about the clients you need to set up for an SAP cluster, see https://documentation.suse.com/sbp/all[].
+
+
+
+== Register Clients to the {SCC}
+
+Register each client with {scc}, using this command as root, specifying the registration number and the email address associated with it:
+
+----
+SUSEConnect -r <Registration_Number> -e <registered_email_address>
+----
+
+You can get your registration number by logging in to {scc} from your browser.
+
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+
+
+
+== Create the Cluster
+
+Every client system must have all the other client systems listed in their [path]``/etc/hosts`` file.
+Open the [path]``/etc/hosts`` file on each client, and add the hostname for each of the other clients.
+
+
+
+== Create a Shared Storage Device
+
+Each of the clients needs to be able to access a shared disk.
+The shared disk can be physical hardware connected by ethernet, or you can set up a virtual disk and access it with iSCSI.
+
+If you use a virtual disk, consider hosting it on a separate system.
+Do not use a client machine to host the shared storage disk.
+
+
+
+== Download the SAP Installation Software
+
+Download the {sles}{nbsp}15 with SAP installation media from {scc}, and save a copy on each clients.
```

Comment:
This paragraph is incorrect. Here we have to state that the users must download and have the **SAP** installation software in the client machines. The files can be stored in a NFS share too.

Here we need to have specific SAP software, that is obtained from the SAP download center (not SUSE).
- For HANA, we need SAP HANA platform (or any similar product).
- For NETWAEVER, S4/HANA things get a more bit difficult, as this requires more software than only one file.

I don't know how we could describe this, but it's not easy.
I don't know how the other SLES4SAP guides mention this part honestly. 
But we have to make sure that this is SAP software, and it's not provided by SUSE